### DISCUSSION TITLE
How to use GitHub?

### DISCUSSION THREAD

**Discussion by AnnoymusScratch** [community/community#31217] [New to GitHub]
I don't know how to check others codes and projects. Please tell me!

**Comment by melvincwng** [comment]
@AnnoymusScratch I recommend you to check out [GitHub Skills](https://skills.github.com/) to learn the basics of GitHub.

Hope this answers your question 😊

**Comment by jfullstackdev** [comment]
first you need to learn the basics of GitHub,

most of the open source have the README file to discuss how to setup their project or you can always ask question at the Issue tab of that repo, or Discussion if that is enabled

**Comment by queenofcorgis** [comment]
Welcome to the world of GitHub :octocat: @AnnoymusScratch - we're glad to have you! @jdevstatic and @melvincwng made some great points, I'd also recommend checking out this [Docs article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests) on using Pull Requests so you can collaborate on code like you mentioned!
